src/utils/tests.js

```diff
@@ -0,0 +1,6 @@
+import { ThemeProvider } from 'styled-components';

```

Comment:
Same here. `utils.tsx`

Reply:
yeah, this one I think i need some other config cause typescript fails at least in visual code. 
does it fails if you change it?

Reply:
I tested on your branch and it's working for me (also vscode). Did you try changing it for ts or tsx?